connect-examples/v2/java_payment/src/main/resources/templates/charge.html

```diff
@@ -9,9 +9,12 @@
 </head>
 
 <body>
-  <p>
-    <span th:text="${payment}"></span>
-  </p>
+  <h1> Charge Result</h1>
```

Comment:
Did this new SDK change the error behavior?

Reply:
I went ahead and added error handling for ApiExceptions and created an error page to better display the error to the user.

Reply:
Nice improvement!